src/components/admin/Billing/CardWrapper.tsx

```diff
@@ -8,35 +8,40 @@ import {
 } from '@mui/material';
 import { defaultBoxShadow, semiTransparentBackground } from 'context/Theme';
 import { HelpCircle } from 'iconoir-react';
-import { FormattedMessage, useIntl } from 'react-intl';
+import { ReactNode } from 'react';
+import { useIntl } from 'react-intl';
 import { BaseComponentProps } from 'types';
-import { TOTAL_CARD_HEIGHT } from 'utils/billing-utils';
 
 interface Props extends BaseComponentProps {
-    messageId: string;
+    message?: string | ReactNode;
```

Comment:
I think that is getting too pedantic for now. This component only shows a single component except for the children so the naming isn't that important. Also, ReactNode does not always mean a component per se.... nice to allow that so that you can use the `FormatMessage` component to pass a translated string.

The `stack` is actually required for the wrapper to work... but I don't remember the scenario. I think it was while I was trying to get this to work well with some vertical growth. I'll remove for now cause that work was abandoned.